requirements.txt

```diff
@@ -1,7 +1,7 @@
 # the requirements.txt file is intended for deployment on streamlit cloud and if the simple container is built
 # note that it is much more restricted in terms of installing third-parties / etc.
 # preferably use the batteries included or simple docker file for local hosting
-streamlit==1.29.0
+streamlit==1.33.0
```

Comment:
The latest is now 1.34, are there issues with that version?

Reply:
no issues with lates